doc/sphinx/user/methods/Stokes-solver/solver.md

```diff
@@ -0,0 +1,110 @@
+#Solver
+
+The Stokes system we are solving is of the
+form
+
+```{math}
+:label: eq:aligned
+    Mx&=\begin{pmatrix}A & B^T \\
+    B & 0
+    \end{pmatrix}
+    \begin{pmatrix} u\\
+    p
+    \end{pmatrix} =
+    \begin{pmatrix}
+    f \\
+    0
+    \end{pmatrix}
+```
+
+. Here, $A$ is the viscous stress matrix,
+$B^T$ comes from integrating 
+grad on pressure space by parts after
+testing with $v$, and $B$ comes from
+integrating div $u$ by parts after testing
+with $q$.
+
+
+We have a right preconditioner of the form
+```{math}
+:label: eq:aligned
+    P^{-1}&=\begin{pmatrix}
+    A^{-1} & A^{-1}B^TS^{-1} \\
+    0      & -S^{-1}
+    \end{pmatrix}
+```
+.
+
+The motivation for this is that
+
+```{math}
+    :label: eq:aligned
+    MP^{-1}&= \begin{pmatrix}
+    AA^{-1} & B^TS^{-1}-B^TS^{-1 } \\
+    BA^{-1} & BA^{-1}B^TS^{-1}
+    \end{pmatrix} \\
+    &\approx
+    \begin{pmatrix}
+    I & 0 \\
+    BA^{-1} & I
+    \end{pmatrix}
+```.
+
+Notice that in the top left block $AA^{-1}=I$ and in the bottom right block $BA^{-1}B^TS^{-1} - 
+we want to choose $S$ such that $S^{-1} \approx BA^{-1}B^T$. Note that 
+this $BA^{-1}B^T$ is often referred to as the Schur Complement.
+
+Pressure Scaling: ASPECT uses dimensionalized units
+and a pressure scaling factor is required in
+order to ensure the iterative
+solver enforces both 
+equations in the Stokes system. 
+The pressure scaling factor
+is computed
+based on viscosity and other parameters.
```

Comment:
From the flow of the argument, it's not clear what the purpose of this paragraph is. Above, you introduce the choice of preconditioner, and then you say that a pressure scaling factor is needed. But then you never actually say anything about the scaling factor again, nor show where in the matrix it appears. You just move on to a different part of the train of thought. The paragraph about the scaling factor sort of hangs in the air, not motivated by what came before nor needed by what comes after.

Reply:
Actually, I see now that it *is* connected to what comes after -- it's just not obvious how. Start the next paragraph with something like this:

"In order to see the need for a scaling factor, recall that an iterative method such as GMRES..."